<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DM3Viewer</class>
 <widget class="QMainWindow" name="DM3Viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>746</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>DM3 Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="DM3Viewer.qrc">
    <normaloff>:/Icons/viewer.svg</normaloff>:/Icons/viewer.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0">
    <item row="0" column="0" colspan="2">
     <widget class="QFrame" name="plotQF">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>827</width>
     <height>36</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_Export">
     <property name="title">
      <string>&amp;Export</string>
     </property>
     <addaction name="action_Export_Data"/>
     <addaction name="action_Export_PNG"/>
     <addaction name="action_Export_JPG"/>
     <addaction name="action_Export_TIF"/>
     <addaction name="action_Export_PDF"/>
    </widget>
    <addaction name="action_Open_DM3"/>
    <addaction name="action_Reload_DM3"/>
    <addaction name="action_Close_DM3"/>
    <addaction name="separator"/>
    <addaction name="menu_Export"/>
    <addaction name="separator"/>
    <addaction name="action_Quit_Viewer"/>
   </widget>
   <widget class="QMenu" name="menuNavigation">
    <property name="title">
     <string>&amp;Navigation</string>
    </property>
    <addaction name="action_First_Image"/>
    <addaction name="action_Previous_Image"/>
    <addaction name="action_Next_Image"/>
    <addaction name="action_Last_Image"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="action_Info"/>
    <addaction name="action_FFT"/>
    <addaction name="separator"/>
    <addaction name="action_Options"/>
    <addaction name="action_Timer"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuNavigation"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open_DM3"/>
   <addaction name="action_Reload_DM3"/>
   <addaction name="action_Close_DM3"/>
   <addaction name="separator"/>
   <addaction name="action_Export_Data"/>
   <addaction name="action_Export_PNG"/>
   <addaction name="action_Export_JPG"/>
   <addaction name="action_Export_TIF"/>
   <addaction name="action_Export_PDF"/>
   <addaction name="separator"/>
   <addaction name="action_First_Image"/>
   <addaction name="action_Previous_Image"/>
   <addaction name="action_Next_Image"/>
   <addaction name="action_Last_Image"/>
   <addaction name="separator"/>
   <addaction name="action_Info"/>
   <addaction name="action_FFT"/>
   <addaction name="separator"/>
   <addaction name="action_Options"/>
   <addaction name="action_Timer"/>
   <addaction name="separator"/>
   <addaction name="action_About"/>
  </widget>
  <action name="action_About">
   <property name="icon">
    <iconset resource="DM3Viewer.qrc">
     <normaloff>:/Icons/help.svg</normaloff>:/Icons/help.svg</iconset>
   </property>
   <property name="text">
    <string>About...</string>
   </property>
   <property name="toolTip">
    <string>About DM3 Viewer</string>
   </property>
   <property name="statusTip">
    <string>See information about DM3 Viewer</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_Options">
   <property name="icon">
    <iconset resource="DM3Viewer.qrc">
     <normaloff>:/Icons/options.svg</normaloff>:/Icons/options.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Options...</string>
   </property>
   <property name="toolTip">
    <string>Edit program options</string>
   </property>
   <property name="statusTip">
    <string>Edit configuration options for DM3 Viewer</string>
   </property>
  </action>
  <action name="action_Open_DM3">
   <property name="icon">
    <iconset resource="DM3Viewer.qrc">
     <normaloff>:/Icons/open.svg</normaloff>:/Icons/open.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Open DM3 file</string>
   </property>
   <property name="toolTip">
    <string>Open DM3 file</string>
   </property>
   <property name="statusTip">
    <string>Select a DM3 file to open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Quit_Viewer">
   <property name="icon">
    <iconset resource="DM3Viewer.qrc">
     <normaloff>:/Icons/quit.svg</normaloff>:/Icons/quit.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit Viewer</string>
   </property>
   <property name="toolTip">
    <string>Quit DM3 Viewer</string>
   </property>
   <property name="statusTip">
    <string>Quit the application</string>
   </property>
  </action>
  <action name="action_First_Image">
   <property name="icon">
    <iconset resource="DM3Viewer.qrc">
     <normaloff>:/Icons/first.svg</normaloff>:/Icons/first.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;First image</string>
   </property>
   <property name="toolTip">
    <string>Move to first image</string>
   </property>
   <property name="statusTip">
    <string>Load the first DM3 image in the current directory</string>
   </property>
   <property name="shortcut">
    <string>Home</string>
   </property>
  </action>
  <action name="action_Previous_Image">
   <property name="icon">
    <iconset resource="DM3Viewer.qrc">
     <normaloff>:/Icons/previous.svg</normaloff>:/Icons/previous.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Previous image</string>
   </property>
   <property name="toolTip">
    <string>Move to previous image</string>
   </property>
   <property name="statusTip">
    <string>Load the previous DM3 image in the current directory</string>
   </property>
   <property name="shortcut">
    <string>PgUp</string>
   </property>
  </action>
  <action name="action_Next_Image">
   <property name="icon">
    <iconset resource="DM3Viewer.qrc">
     <normaloff>:/Icons/next.svg</normaloff>:/Icons/next.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Next image</string>
   </property>
   <property name="toolTip">
    <string>Move to next image</string>
   </property>
   <property name="statusTip">
    <string>Load the next DM3 image in the current directory</string>
   </property>
   <property name="shortcut">
    <string>PgDown</string>
   </property>
  </action>
  <action name="action_Last_Image">
   <property name="icon">
    <iconset resource="DM3Viewer.qrc">
     <normaloff>:/Icons/last.svg</normaloff>:/Icons/last.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Last image</string>
   </property>
   <property name="toolTip">
    <string>Move to last image</string>
   </property>
   <property name="statusTip">
    <string>Load the last DM3 image in the current directory</string>
   </property>
   <property name="shortcut">
    <string>End</string>
   </property>
  </action>
  <action name="action_Timer">
   <property name="icon">
    <iconset resource="DM3Viewer.qrc">
     <normaloff>:/Icons/timer.svg</normaloff>:/Icons/timer.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Timer</string>
   </property>
   <property name="toolTip">
    <string>Timer</string>
   </property>
   <property name="statusTip">
    <string>Iterate automatically over all the DM3 images in the current directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_Export_Data">
   <property name="icon">
    <iconset resource="DM3Viewer.qrc">
     <normaloff>:/Icons/save_dat.svg</normaloff>:/Icons/save_dat.svg</iconset>
   </property>
   <property name="text">
    <string>Export &amp;data...</string>
   </property>
   <property name="toolTip">
    <string>Export data as text</string>
   </property>
   <property name="statusTip">
    <string>Export the data in the current DM3 image as text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_Export_PNG">
   <property name="icon">
    <iconset resource="DM3Viewer.qrc">
     <normaloff>:/Icons/save_png.svg</normaloff>:/Icons/save_png.svg</iconset>
   </property>
   <property name="text">
    <string>Export as &amp;PNG...</string>
   </property>
   <property name="toolTip">
    <string>Export image as PNG</string>
   </property>
   <property name="statusTip">
    <string>Export the data in the current DM3 image as PNG</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_Export_JPG">
   <property name="icon">
    <iconset resource="DM3Viewer.qrc">
     <normaloff>:/Icons/save_jpg.svg</normaloff>:/Icons/save_jpg.svg</iconset>
   </property>
   <property name="text">
    <string>Export as &amp;JPG...</string>
   </property>
   <property name="toolTip">
    <string>Export image as JPG</string>
   </property>
   <property name="statusTip">
    <string>Export the data in the current DM3 image as JPG</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="action_Export_TIF">
   <property name="icon">
    <iconset resource="DM3Viewer.qrc">
     <normaloff>:/Icons/save_tif.svg</normaloff>:/Icons/save_tif.svg</iconset>
   </property>
   <property name="text">
    <string>Export as &amp;TIF...</string>
   </property>
   <property name="toolTip">
    <string>Export image as TIF</string>
   </property>
   <property name="statusTip">
    <string>Export the data in the current DM3 image as TIF</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_Export_PDF">
   <property name="icon">
    <iconset resource="DM3Viewer.qrc">
     <normaloff>:/Icons/save_pdf.svg</normaloff>:/Icons/save_pdf.svg</iconset>
   </property>
   <property name="text">
    <string>Export as PD&amp;F</string>
   </property>
   <property name="toolTip">
    <string>Export image as PDF</string>
   </property>
   <property name="statusTip">
    <string>Export the data in the current DM3 image as PDF</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_Info">
   <property name="icon">
    <iconset resource="DM3Viewer.qrc">
     <normaloff>:/Icons/info.svg</normaloff>:/Icons/info.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Info...</string>
   </property>
   <property name="toolTip">
    <string>Image info</string>
   </property>
   <property name="statusTip">
    <string>Show the info in the metadata of the current DM3 image</string>
   </property>
  </action>
  <action name="action_Close_DM3">
   <property name="icon">
    <iconset resource="DM3Viewer.qrc">
     <normaloff>:/Icons/close.svg</normaloff>:/Icons/close.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Close DM3 file</string>
   </property>
   <property name="statusTip">
    <string>Close the current DM3 file</string>
   </property>
  </action>
  <action name="action_Reload_DM3">
   <property name="icon">
    <iconset resource="DM3Viewer.qrc">
     <normaloff>:/Icons/reload.svg</normaloff>:/Icons/reload.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Reload DM3 file</string>
   </property>
   <property name="statusTip">
    <string>Reload the current DM3 file</string>
   </property>
  </action>
  <action name="action_FFT">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="DM3Viewer.qrc">
     <normaloff>:/Icons/fft.svg</normaloff>:/Icons/fft.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;FFT</string>
   </property>
   <property name="toolTip">
    <string>Fast Fourier Transform</string>
   </property>
   <property name="statusTip">
    <string>Calculate the Fast Fourier Transform of the current DM3 file</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="DM3Viewer.qrc"/>
 </resources>
 <connections/>
</ui>
